tests/objects/omitBy.test.js

```diff
@@ -0,0 +1,32 @@
+const omitBy = require('../../src/objects/omitBy');
+
+describe('omitBy', () => {
+  test("Creates an object composed of the own and inherited enumerable string keyed properties of object that predicate doesn't return truthy for.", () => {
+    var object = { 'a': 1, 'b': '2', 'c': 3 };
+    var objectSec = { 'a': 'aaa', 'b': '2', 'c': 3 , 'd': '44'};
+
+    let funcNotNumber= (obj) => {
```

Comment:
please start function naming with "action verb": return.., collect..., calculate..., get..., set.. and etc.